<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRelativeAircraftPosition</class>
 <widget class="QFrame" name="CRelativeAircraftPosition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Relative coordinate</string>
  </property>
  <layout class="QGridLayout" name="gl_RelativeAircraftPosition">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_Distance">
     <property name="text">
      <string>distance:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lbl_RelativeBearing">
     <property name="text">
      <string>rel.bearing</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="sb_RelativeBearing">
     <property name="suffix">
      <string> deg.</string>
     </property>
     <property name="maximum">
      <number>359</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="sb_Distance">
     <property name="suffix">
      <string> m</string>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::DefaultStepType</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="le_Info">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
